StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/ElementDocumentationBase.cs

```diff
@@ -235,28 +235,28 @@ private void HandleDeclaration(SyntaxNodeAnalysisContext context, StyleCopSettin
                     .Where(x => !string.Equals(x.GetName()?.ToString(), XmlCommentHelper.IncludeXmlTag, StringComparison.Ordinal))
                 : documentation.Content.GetXmlElements(this.matchElementName);
 
-            if (!matchingXmlElements.Any())
+            var isInheritingDocumentation =
+                documentation.Content.GetFirstXmlElement(XmlCommentHelper.IncludeXmlTag) is object;
```

Comment:
I would prefer hasIncludedDocumentation, as the current name could cause confusion with the <inheritdoc/> tag.

Reply:
Thanks! I have commited the changes to 64de3245af819d2a6c716a919fc47e11c2c24a0a.